setup.py

```diff
@@ -26,7 +26,7 @@
 
         # Specify the Python versions you support here. In particular, ensure
         # that you indicate whether you support Python 2, Python 3 or both.
-        'Programming Language :: Python :: 3.8, 3.9, 3.10',
+        'Programming Language :: Python :: 3.9, 3.10, 3.11',
```

Comment:
Did you actually test all of these versions? Including a full conda install with these Python versions, from scratch?

Reply:
grblblb, yes I just did.